src/app/games/lasca/LascaControlAndDomination.ts

```diff
@@ -1,24 +1,14 @@
 import { BoardValue } from 'src/app/jscaip/BoardValue';
-import { GameStatus } from 'src/app/jscaip/GameStatus';
-import { MGPNode } from 'src/app/jscaip/MGPNode';
+import { Minimax } from 'src/app/jscaip/Minimax';
 import { Player } from 'src/app/jscaip/Player';
-import { LascaControlMinimax } from './LascaControlMinimax';
+import { LascaControlHeuristic, LascaMoveGenerator } from './LascaControlMinimax';
 import { LascaMove } from './LascaMove';
-import { LascaRules } from './LascaRules';
+import { LascaNode, LascaRules } from './LascaRules';
 import { LascaStack, LascaState } from './LascaState';
 
-class LascaNode extends MGPNode<LascaRules, LascaMove, LascaState> {}
+export class LascaControlAndDominationHeuristic extends LascaControlHeuristic {
```

Comment:
add minimax to the name

Reply:
aussi: Y aura jamais plusieurs MCTS right ? Donc on peux ne pas ajouter "minimax" aux minimax ?

Reply:
c'est une heuristique, pas un minimax, donc oui, pas besoin d'ajouter minimax dans le nom de l'heuristique je dirais. Ai-je bien compris ?

Reply:
Oui, donc quand les IA c'est ["Chaussettes Minimax", "Pantalon Minimax", "MCTS"] on peut enlever Minimax !
Chaud renommage !